src/main/java/gift/domain/order/TemplateObject.java

```diff
@@ -24,4 +17,11 @@ public TemplateObject(String text) {
             ),
             "check button");
     }
+
+    public record Link(
+        String web_url,
```

Comment:
자바 컨벤션에 맞게 변수명 작성하는 것이 좋을 것 같네요! 그러면 카카오 스펙과 맞지 않아 에러가 발생할텐데, 추가적인 해결이 필요할꺼에요!
```suggestion
        String webUrl,
        String mobileWebUrl
```

Reply:
objectMapper에서 json 문자열로 변환 시 따로 카멜 케이스를 스네이크 케이스로 바꿔주지 않아서 변수명을 직접 스네이크 케이스로 했었습니다.
objectMapper 가 카멜 케이스를 스네이크 케이스로 바꿔주도록 하는 속성값을 사용하는 것으로 변경했습니다!